src/unstract/sdk/file_storage/helper.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import os
+
+import fsspec
+from google.cloud import storage
+
+from unstract.sdk.exceptions import FileStorageError
+from unstract.sdk.file_storage.constants import GCS
+from unstract.sdk.file_storage.fs_type import FileStorageType
+
+logger = logging.getLogger(__name__)
+
+
+class FileStorageHelper:
+    @staticmethod
+    def file_storage_init(type: FileStorageType, credentials):
+        if type == FileStorageType.GCS:
+            return FileStorageHelper.gcs_init(credentials)
+        elif type == FileStorageType.Local:
+            return FileStorageHelper.local_file_system_init()
+
+    @staticmethod
+    def gcs_init(credentials):
+        try:
+            os.environ[GCS.CREDS] = credentials[GCS.CREDS]
+            storage_client = storage.Client()
+
+            fs = fsspec.filesystem(
+                protocol=FileStorageType.GCS.value,
+                project=credentials[GCS.PROJECT_ID],
+                client=storage_client,
+            )
+            logger.info(f"Connected to {FileStorageType.GCS.value} file system")
+        except Exception as e:
+            logger.error(
```

Comment:
@gaya3-zipstack also please add the exc_info and stack_info in logger.error so that the entire error trace will be printed. I think that would be needed for debugging.

Reply:
@hari-kuriakose @gaya3-zipstack @ritwik-g do we wish to log the error with the traceback in the SDK? If we're treating it as a library, I suggest we just raise the exception alone and let the downstream handle the logging. This can help avoid repeated tracebacks
- We've already added some logging middlewares in most services which log the traceback itself. 
- This might be missing in our celery workers alone which can be supported as well with a try-except on the shared task